src/pages/index.js

```diff
@@ -123,29 +266,89 @@ addButton.addEventListener("click", () => {
 
     // Open the add card form
     addFormPopup.open();
-});
-
-// Set add form event listeners
-addFormPopup.setEventListeners();
 
+    // Set add form event listeners
+    addFormPopup.setEventListeners();
+});
 
 /* -------------------------------------------------------------------------- */
 /*                             Profile Information                            */
 /* -------------------------------------------------------------------------- */
 
-// Create the user info instance
-const userInfo = new UserInfo(selectors.profileName, selectors.profileProfession);
-
 // Create the edit form instance
 const editFormPopup = new PopupWithForm(selectors.editFormPopup, (values) => {
 
-    // Add the form's input to the profile section
-    userInfo.setUserInfo(values.name, values.profession);
+    // Render loading status
+    renderLoading(true, profileSaveButton, 'Saving...');
+
+    // Update the user info in the server
+    api.updateUserinfo(values.name, values.profession)
+    .then(values => {
+
+        // Add the form's input to the profile section
+        userInfo.setUserInfo(values.name, values.about);
+    })
+    .then(() => {
```

Comment:
There is no need to have two `then` blocks

Reply:
Great! That's removed.